Sources/KlaviyoSwift/KlaviyoEnvironment.swift

```diff
@@ -23,20 +25,45 @@ let decoder = {
     return decoder
 }()
 
+private let reachabilityService = Reachability(hostname: "a.klaviyo.com")
```

Comment:
Should we have only 1 reference for this base url?

Reply:
I mean, only one place where the base url is defined, and reference that here

Reply:
Oh technically this should be refreshed when the url changes...but that would only affect sdks running against locals. Are you more referring to having a constant for this?

Reply:
Yeah i think you had introduced a way to have this as a constant, but overridable in DEBUG mode, so I was thinking about that.

Reply:
Yah I was actually gonna deprecate that in favor of the URLProtocol method but didn't get to that yet.

Reply:
Cool, this is definitely a nit, its still easy enough to change the url while developing locally

Reply:
I think actually I can turn this into a function that creates the service.